doc/install.txt

```diff
@@ -67,7 +67,9 @@ First, install the required software.
    ClamAv from http://www.clamav.net/
 
 7. *Optional, for GNOME proxy setting parsing:*
-    Python Gtk from http://www.pygtk.org/downloads.html
+    PyGObject, GIO and its repository file.
+    Best installed from your distribution e.g. ``python3-gi`` and
+    ``libgirepository1.0-dev``
```

Comment:
Is libgirepository1.0-dev needed?

> This package contains the headers for the C library for handling the
> introspection data.

AFAIU all that pygobject-introspection needs is typelib files, packaged in gir1.2-glib-2.0, which is pulled in by python3-gi.

Reply:
Oops yes, thanks. It is me that likes reading the gir files not PyGObject.